### ISSUE TITLE
how to disconnect?

### ISSUE THREAD

**Issue by blodone** [jonashagstedt/swampdragon#123]
Is there any way to disconnect a swampdragon connection?

**Comment by xxxkilldogxxx** [comment]
On the front end I believe their is a call for it?

**Comment by blodone** [comment]
yes i have seen this in the legacy version but i didn't recognize it for the angular version... because if i use swampdragon-auth and the server gets restarted / disconnected i would like to recognize that in the frontend and re-auth / re-subscribe with the correct auth again, but i did not find a way to do this
